src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -205,6 +218,90 @@ public WindowState WindowState
             }
         }
 
+        private void MarkFullscreen(bool fullscreen)
```

Comment:
`private` functions placed in completely random location.

Reply:
moved.